packages/utils/isValidAddress.ts

```diff
@@ -18,14 +18,18 @@ export function isValidValidatorAddress(
   chainPrefix: string
 ): boolean {
   const bech32Regex = /^[a-km-zA-HJ-NP-Z0-9]{46}$/im
+  // Some validators may be run by DAOs and have contract addresses
+  const bech32ContractRegex = /^[a-km-zA-HJ-NP-Z0-9]{66}$/im
```

Comment:
can you add a comment about why this uses `66` and the function below this uses `59` for its contract regex? (might be good to point out the difference of `valoper` here)